cmd/yurthub/app/config/config.go

```diff
@@ -72,28 +76,33 @@ func Complete(options *options.YurtHubOptions) (*YurtHubConfiguration, error) {
 
 	hubServerAddr := net.JoinHostPort(options.YurtHubHost, options.YurtHubPort)
 	proxyServerAddr := net.JoinHostPort(options.YurtHubHost, options.YurtHubProxyPort)
+	proxySecureServerAddr := net.JoinHostPort(options.YurtHubHost, options.YurtHubProxySecurePort)
```

Comment:
yurthub's https also need to listen on `HubAgentDummyIfIP` address

Reply:
The latest code has been commited. Yurthub's https also listens on HubAgentDummyIfIP address